data/quarg/quarg news.txt

```diff
@@ -442,3 +442,67 @@ news "student on quarg planets"
 			"."
 		word
 			`"`
+
+news "quarg anywhere"
+	location
+		attributes "quarg" "human quarg" "hai quarg"
+		not attributes "korath quarg" "gegno quarg"
+	name
+		word
+			"Quarg"
+	message
+		word
+			`"Yes human? Do you have any questions?"`
+			`"Hello human. Are you enjoying your time on our world? We are glad to receive visitors."`
```

Comment:
```suggestion
			`"Yes, human? Do you have any questions?"`
			`"Hello, human. Are you enjoying your time on our world? We are glad to receive visitors."`
```
It seems to me that the Quarg would be a patient species and that it might be reflected in them pausing.